ci/test_runner.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -euo pipefail
+
+# Conjur test runner configuration environment:
+# KUBE_NODE_TAG:   Docker tag for the Kubernetes node image. This
+#                  determines the Kubernetes version of the KinD cluster.
+#                  Available versions can be found here:
+#                      https://github.com/kubernetes-sigs/kind/releases
+# CLUSTER_NAME:  Name of KinD cluster to be created.
+# MULTI_NODE:    If set to "true", KinD cluster is created with a master
+#                node and 2 worker nodes. Otherwise, a single-node cluster
+#                is created.
+KUBE_NODE_TAG=${KUBE_NODE_TAG:-v1.17.5}
+CLUSTER_NAME=${CLUSTER_NAME:-v1.17.5}
+MULTI_NODE=${MULTI_NODE:-false}
+IPV6_CLUSTER=${IPV6_CLUSTER:-false}
+
+echo "=============================="
+echo "Test Runner Configuration:"
+echo "KUBE_NODE_TAG: $KUBE_NODE_TAG"
+echo "CLUSTER_NAME:  $CLUSTER_NAME"
+echo "MULTI_NODE:    $MULTI_NODE"
+echo "IPV6_CLUSTER:  $IPV6_CLUSTER"
+echo "=============================="
+
+function delete_cluster {
+    echo "=============================="
+    echo "Deleting Kubernetes cluster..."
+    echo "=============================="
+    kind delete cluster --name "${CLUSTER_NAME}"
+}
+
+echo "==================================="
+echo "Creating KinD configuration file..."
+echo "==================================="
+KUBE_CONFIG="$(pwd)/kind_config.yaml"
+cat > $KUBE_CONFIG <<EOT
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  image: "kindest/node:$KUBE_NODE_TAG"
+EOT
```

Comment:
Add a newline at the end of each `EOT` and/or at the beginning of `if`s - it's hard to read this as a single block